modules/lemmaBidAdapter.md

```diff
@@ -0,0 +1,67 @@
+# Overview
+
+```
+Module Name:  Lemma Bid Adapter
+Module Type:  Bidder Adapter
+Maintainer: lemmadev@lemmatechnologies.com
+```
+
+# Description
+
+Connects to Lemma exchange for bids.
+Lemma bid adapter supports Video, Banner formats.
+
+# Sample Banner Ad Unit: For Publishers
+```
+var adUnits = [{
+  code: 'div-lemma-ad-1',
+  mediaTypes: {
+    banner: {
+      sizes: [[300, 250], [300, 600]],  // required
+    }
+  },
+  // Replace this object to test a new Adapter!
+  bids: [{
+    bidder: 'lemma',
+    params: {
+      pubId: '1', // required
```

Comment:
Per `isBidRequestValid` checks, this should be 1 instead of '1'

Reply:
I have corrected parameter value.